src/styles/blocks/container.scss

```diff
@@ -0,0 +1,5 @@
+
+.container {
+  @extend %flex-center;
+  height: 100vh;
```

Comment:
```suggestion
```
There is no need to hardcode height. Let the height of the elements determines the height of the container